Documentation~/TableOfContents.md

```diff
@@ -109,8 +111,8 @@
 * [ProBuilder Preferences](preferences)
 * [Troubleshooting](faq)
 * [Experimental features](experimental)
-	* [Create and edit Bezier shapes](workflow-create-bezier)
-	* [Perform Boolean operations](boolean)
-	* [Store meshes as assets](meshes-as-assets)
+	* [Bezier Shape tool](workflow-create-bezier)
```

Comment:
I have a feeling I don't want this change. Please don't merge until I've figured it out.